Feat: Refactors section grid (#1779)
vtex/faststore: apps/site/pages/docs/customizing/pages.mdx

            value: product.offers.offers[0].price,
            listPrice: product.offers.offers[0].listPrice,
            formatter: useFormattedPrice,
          }}
        />
      </ProductCard>
    </li>
  </ul>
)

<Tabs items={['Tiles', 'Banner', 'Code']} defaultIndex="0">
  <Tab>
    <OverviewSection direction="column" containerStyle={{ padding: '0' }}>
      <section class="section layout__section | nx-section-layout">
        {TilesTemplate}
      </section>
    </OverviewSection>
  </Tab>
  <Tab>
    <OverviewSection direction="column" containerStyle={{ padding: '0' }}>
      <section class="section layout__section | nx-section-layout">
        <Banner colorVariant="light">
          <BannerContent
            title="Explore more about our amazing products and discover a world of possibilities."
            caption="All the amazing products from the brands we partner with."
            link="/"
            linkText="See more"
          />
        </Banner>
      </section>
    </OverviewSection>
  </Tab>
  <Tab>
    ```scss
    // This is already set as default
    [data-fs-content] { @include layout-content; }
    ```
  </Tab>
</Tabs>

### Content centered + Colored bg

> If you want to add a **colored background** with **full width** and mantain the **content centered**, you should add `@include layout-colored` to your desired section.

Reviewer: ```suggestion
To display a *colored background* with *full width* and maintain the *content centered*, add `@include layout-colored` to your desired section.
```

<Tabs items={['Tiles', 'Banner', 'Code']} defaultIndex="0">
  <Tab>
    <OverviewSection containerStyle={{ padding: '0' }}>
      <section class="section layout__section | nx-section-layout nx-section-layout-colored">
        {TilesTemplate}
      </section>
    </OverviewSection>
  </Tab>
  <Tab>
    <OverviewSection direction="column" containerStyle={{ padding: '0' }}>
      <section class="section layout__section | nx-section-layout nx-section-layout-colored">
        <Banner colorVariant="light">
          <BannerContent
            title="Explore more about our amazing products and discover a world of possibilities."
            caption="All the amazing products from the brands we partner with."
            link="/"
            linkText="See more"
          />
        </Banner>
      </section>
    </OverviewSection>
  </Tab>
  <Tab>
    ```scss
    // Add this to your custom-theme.scss
    .section {
      [data-fs-tiles] {
        @include layout-colored(var(--fs-color-main-2));
      }

      [data-fs-banner] {
        @include layout-colored(var(--fs-color-main-2));
      }
    }
    ```

  </Tab>
</Tabs>

### Content centered + Colored bg + Section centered

> If you want to add a **colored background** and **centering** the `.section`, you should add `@include layout-colored` and `@include layout-content` to your desired section.

Reviewer: ```suggestion
To add a *colored background* and *centering* the `.section`, add `@include layout-colored` and `@include layout-content` to your desired section.
```

  <Tab>
    <OverviewSection containerStyle={{ padding: '0' }}>
      <section class="section layout__section layout__content | nx-section-layout nx-section-layout-colored">
        {TilesTemplate}
      </section>
    </OverviewSection>
  </Tab>
  <Tab>
    <OverviewSection direction="column" containerStyle={{ padding: '0' }}>
      <section class="section layout__section layout__content | nx-section-layout nx-section-layout-colored">
        <Banner colorVariant="light">
          <BannerContent
            title="Explore more about our amazing products and discover a world of possibilities."
            caption="All the amazing products from the brands we partner with."
            link="/"
            linkText="See more"
          />
        </Banner>
      </section>
    </OverviewSection>
  </Tab>
  <Tab>
    ```scss
    // Add this to your custom-theme.scss
    .section {
      [data-fs-tiles] {
        @include layout-colored(var(--fs-color-main-2));
        @include layout-content;
      }

      [data-fs-banner] {
        @include layout-colored(var(--fs-color-main-2));
        @include layout-content;
      }
    }
    ```

  </Tab>
</Tabs>

### Full width layout

> If for some reason you want to use your section's content with **full width layout** you'll need to change it's content to include the `@mixin layout-content-full`.

Reviewer: ```suggestion
To use your store section's content with a *full-width layout* you'll need to change its content to include the `@mixin layout-content-full`.
```
